packages/website/src/vendor/sandbox.d.ts

```diff
@@ -1,10 +1,10 @@
-import { TypeScriptWorker } from './tsWorker';
```

Comment:
this file is downloaded from ts, we should not modify it.

Reply:
The changes I made:
- switch `any` for `unknown`
- replace inline type import with type-only import declarations
- deduplicate code
- apply formatter

Functionally the code is the ecactly same - it just adheres to our code style now.

Reply:
you should update tools/generate-website-dts.ts, to apply those changes than :)

Reply:
I didn't realise they were generated!